imap_processing/ultra/l1b/cullingmask.py

```diff
@@ -25,14 +26,31 @@ def calculate_cullingmask(
     cullingmask_dataset : xarray.Dataset
         Dataset containing the data.
     """
-    cullingmask_dict = {}
-    cullingmask_dict["spin_number"] = extendedspin_dataset["spin_number"]
-    cullingmask_dict["energy_bin_geometric_mean"] = extendedspin_dataset[
-        "energy_bin_geometric_mean"
-    ]
-
-    # TODO: add more data to cullingmask_dict.
-
-    cullingmask_dataset = create_dataset(cullingmask_dict, name, "l1b", data_version)
+    # If the spin rate was too high or low then the spin should be thrown out.
+    # If the rates at any energy level are too high then throw out the entire spin.
+    mask = (
```

Comment:
Suggest putting "good" or something similar to indicate this is the good values you are selecting (not the bad values which I think of as masked).
```suggestion
    good_mask = (
```